Makefile

```diff
@@ -0,0 +1,40 @@
+init:
+	git submodule update --init --remote
+
+update:
+	git submodule update --remote
+
+run-saleor:
+	cd saleor && docker-compose up -d
+
+run-storefront:
+	cd saleor-storefront && docker container run -d -p 3000:3000 --mount source=$(shell pwd)/saleor-storefront/src,target=/app/src,type=bind,consistency=cached saleor-storefront
+
+run-dashboard:
+	cd saleor-dashboard && docker container run -d -p 9000:9000 --mount source=$(shell pwd)/saleor-dashboard/src,target=/app/src,type=bind,consistency=cached saleor-dashboard
```

Comment:
Why not move the Docker Compose config to this repository (and refer to `Dockerfile`s in subdirectories)?

Reply:
@patrys thought about having compose file here that would extend the Saleor compose file but it is a docker limitation that services having `depends_on` [cannot be extended](https://github.com/docker/compose/issues/3220). Moving compose here (contents) on the other hand would require to update it each time it's being updated in Saleor. That's why I just left it as is.

Reply:
What if we moved it instead of having two copies that need to be kept in sync?

Reply:
about moving compose from saleor to saleor-platform - if saleor repo doesn't have the compose file you won't be able to run it, if you for ex. you choose not to use entire platform (just the backend which you cloned manually from saleor repo).

So there are three options I can think of:
1) remove compose from saleor, put it here but it has drawbacks as mentioned
2) copy saleors compose here (with modifications like context to make it work), also put extra services there (storefront & dashboard) - we can get rid off makefile and windows batch script. Drawback is that we will have to update it each time it gets changed in saleor.
3) don't put any compose here, leave it in saleor, have makefile handle stuff (as currently implemented)

_keep in mind only saleor uses compose, storefront and dashboard does not, they just have dockerfiles_

What do you guys think? @patrys  @maarcingebala

Reply:
The compose in Saleor used to give you an entire platform. It's no longer the case because we moved parts of it to separate repositories. I think it's easier for advanced users to disable certain services in a combined compose file than it is for new users to deal with makefiles.